Formula/g/gitsign.rb

```diff
@@ -23,6 +23,8 @@ def install
       -X github.com/sigstore/gitsign/pkg/version.gitVersion=#{version}
     ]
     system "go", "build", *std_go_args(ldflags:)
+    system "go", "build", *std_go_args(ldflags:, output: bin/"gitsign-credential-cache"), \
+      "./cmd/gitsign-credential-cache"
```

Comment:
I dont think you need the trailing `\`
```suggestion
    system "go", "build", *std_go_args(ldflags:, output: bin/"gitsign-credential-cache"),
      "./cmd/gitsign-credential-cache"
```

Reply:
also you might want to generate the completions for it as well?

Reply:
I had considered the completions but the credential cache only defines two flags (one of which is `--version`), so I don't believe there would be much benefit.

Reply:
> I dont think you need the trailing `\`

Thanks, I'm new to writing formulae.